geoportailv3/static/js/catalog/catalogdirective.js

```diff
@@ -27,7 +27,8 @@ app.catalogDirective = function() {
   return {
     restrict: 'E',
     scope: {
-      'map': '=appCatalogMap'
+      'map': '=appCatalogMap',
+      'currentTheme': '=appCurrentTheme'
```

Comment:
I'd suggest renaming the attribute to `appCatalogTheme` to match the convention we use elsewhere. The convention is `appDirectiveAttribute` (`app-directive-attribute`).